### PR TITLE
[le10] use a larger console font on HiDPI displays

### PR DESCRIPTION
**Pull request by HiassofT**
Console output (eg during an update or if a filesystem error occurred) is almost unreadable on 4k screens.

Use the terminus 16x32 font instead of the default built-in 8x16 font on display with more than 1080 lines. This is done very eary during init, after load_modules, so that errors from check_disks and subsequent init steps can be properly seen.

Runtime tested with RPi4 connected to 4k TV

### CODE REVIEW

**Comment by HiassofT** [comment]
picture of current console output:
update-std

picture of console output with this PR
update-new

**Comment by HiassofT** [comment]
added a commit to make the large-font resolution limit configurable - thanks @MilhouseVH fo the suggestion!

use eg `bigfont=720` on cmdline.txt to use the larger font on vertical resolutions greater than 720 lines

File: packages/sysutils/terminus-font/package.mk [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+PKG_LONGDESC="This package contains the Terminus Font"
+PKG_TOOLCHAIN="manual"
+
+pre_configure_init() {
+  cd $PKG_BUILD
+  rm -rf .${TARGET_NAME}-${TARGET}
+}
+
+configure_init() {
+  ./configure INT=${TOOLCHAIN}/bin/python3
```
Comment (MilhouseVH) [inline packages/sysutils/terminus-font/package.mk]: Presumably this has a Python3 package dependency?

Comment (MilhouseVH) [reply packages/sysutils/terminus-font/package.mk]: Python3:host

**Pull request merged**